functions/src/models/organizations/index.ts

```diff
@@ -0,0 +1,63 @@
+import { FirestoreDataConverter } from '@google-cloud/firestore';
+import { IsArray, IsNotEmpty, IsString } from 'class-validator';
+import { firestore } from 'firebase-admin';
+import DocumentData = firestore.DocumentData;
+
+export enum OrganizationType {
+  healthCare = 'health care',
+  socialServices = 'social services',
```

Comment:
i think for enum values, it's better to not have spaces. perhaps use underscores instead?

```suggestion
  healthCare = 'health_care',
  socialServices = 'social_services',
```

Reply:
In the code we won’t use the `string with spaces` so this is more for the front end team to be able to render? But they could also use a converter for the enum so.... 😅